Shared/LaunchArguments.swift

```diff
@@ -12,6 +12,7 @@ public struct LaunchArguments {
     public static let SkipETPCoverSheet = "FIREFOX_SKIP_ETP_COVER_SHEET"
     public static let ClearProfile = "FIREFOX_CLEAR_PROFILE"
     public static let StageServer = "FIREFOX_USE_STAGE_SERVER"
+    public static let ChinaServer = "FIREFOX_USE_CHINA_SERVER"
```

Comment:
Maybe `FxAChinaServer` (@nbhasin2 thoughts?)

Reply:
`FxAChinaServer` sounds better

Reply:
Thanks I will modify that :)